nbb/package-lock.json

```diff
@@ -0,0 +1,347 @@
+{
```

Comment:
This is a lock file from npm, but we're calling `yarn install`, leading to this warning:
> warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.

Reply:
There is no `npx` in yarn. Why is nextjournal using yarn instead of the default node tooling?

Reply:
It seems `yarn nbb` also works. TIL.